src/foam/blob/Blob.js

```diff
@@ -47,60 +87,32 @@ foam.INTERFACE({
     {
       name: 'put',
       returns: 'Promise',
-      args: [
-        {
-          name: 'blob',
-          of: 'foam.blob.Blob'
-        }
-      ]
+      args: [ 'blob' ]
```

Comment:
I'd rather keep the additional information unless it's causing problems.  In the future we will use it to remove the need to specify javaType in the args

Reply:
Okay, I've added it back